app/views/categories/show.html.haml

```diff
@@ -1,5 +1,6 @@
 - content_for(:rel_next_prev_link_tags) { rel_next_prev_link_tags(@posts) }
 - breadcrumb :category, @category
+- breadcrumb :category_page, @category if params[:page]
```

Comment:
I prefer to use params on only controller.
`@posts` has `current_page` method, so why don't you use `@posts.current_page` instead of `params`?